controller_test.go

```diff
@@ -95,6 +96,8 @@ func TestNodeDeleted(t *testing.T) {
 		DeleteWaitChan: make(chan struct{}),
 	}
 
+	logger, _ := micrologger.New(micrologger.DefaultConfig())
```

Comment:
Not a blocker, and not a big deal at all here, but I would recommend checking errors, even in tests, you can catch some weirdnesses.